test/fixtures/server-test/webpack.client.server.config.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 module.exports = [{
+  mode: 'development',
```

Comment:
gotta remove all these `mode` additions. i'll be wiring up dual testing for v3 and v4 and this will throw on v3. instead, use a conditional to add this only when running with v4